src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import Completed from './components/status.js' ;
+
+
+const ul = document.querySelectorAll("ul");
+const todos = JSON.parse(localStorage.getItem('todo')) || [
+  {
+    description: "Take a shower",
+    completed: false,
+    //index: 1,
+  },
+  {
+    description:"Read my new book",
+    completed: false,
+    //index: 2,
+  },
+  {
+    description: "Wash asdfasdfasdfasdf dishes",
+    completed: true,
+    //index: 3,
+  },
+  {
+    description: "Go at the supermarket",
+    completed: true,
+    //index: 4,
+  },
+];
+
+const displayTodos = () => {
+  todos.forEach((item, index) => {
+    console.log(item);
```

Comment:
### JavaScript requirements

Let's remove `console.log`

![image](https://user-images.githubusercontent.com/31897434/130103844-a4ff8a42-c4ca-4918-b071-0e06e4a514ed.png)

Reply:
Thank you for review my project. I improved these requested changes.